frontend/src/custom-hooks/useAuthzStakingExecHelper.tsx

```diff
@@ -74,18 +84,34 @@ const useAuthzStakingExecHelper = () => {
     const address = convertAddress(data.chainID, data.granter);
     const grants: Authorization[] =
       authzChains?.[data.chainID]?.GrantsToMeAddressMapping?.[address] || [];
+    let isExpired = false;
     const haveGrant = grants.some((grant) => {
-      return (
-        // todo: stake Authorization
+      if (
         (grant.authorization['@type'] ===
           '/cosmos.authz.v1beta1.GenericAuthorization' &&
```

Comment:
Defining these messages as constants in a separate file would be better. This will make it easier to modify them in the future if needed.